README.md

```diff
@@ -0,0 +1,43 @@
+# Weather Server
+
+A Go based learning project that creates an HTTP server and queryable api endpoint.
+
+```bash
+./weather-server -port 9011 -log verbose
+{Clouds few clouds 81.86 Seattle}
+```
+
+## Usage
+* Prior to use, install [Golang](https://golang.org/doc/install)
```

Comment:
We could provide some other ways to install the program, couldn't we? One might be a `curl XXX |sh` self-installer, another might be as a public Docker image. You might like to experiment with these (perhaps creating them automatically via GitHub Actions, for example).

Reply:
I'm still looking into this.  I have a m1 mac so trying to see if GitHub Actions will support this platform.